### PR TITLE
Fixed tests that used ESM

### PR DESCRIPTION
**Pull request by shanejonas**

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ module.exports = {
 
   // Indicates which provider should be used to instrument code for coverage
   coverageProvider: 'babel',
-  rootDir: './src',
+  // rootDir: './src',
```
Comment (BelfordZ) [inline jest.config.js:37]: rm?

File: jest.config.js [hunk 2]
```diff
@@ -134,9 +135,7 @@ module.exports = {
   // rootDir: undefined,
 
   // A list of paths to directories that Jest should use to search for files in
-  // roots: [
-  //   "<rootDir>"
-  // ],
+  // roots: [],
```
Comment (BelfordZ) [inline jest.config.js:138]: rm?

**Review by BelfordZ — DISMISSED** [review]

File: package.json [hunk 3]
```diff
@@ -28,7 +28,7 @@
     "lint:fix": "yarn lint:eslint --fix && yarn lint:constraints --fix && yarn lint:misc --write && yarn lint:dependencies && yarn lint:changelog",
     "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore --no-error-on-unmatched-pattern",
     "prepack": "./scripts/prepack.sh",
-    "test": "jest && jest-it-up",
+    "test": "jest",
```
Comment (shanejonas) [inline package.json:31]:
```suggestion
    "test": "jest && jest-it-up",
```

**Review by BelfordZ — DISMISSED** [review]

**Comment by shanejonas** [comment]
@SocketSecurity ignore-all

**Review by BelfordZ — APPROVED** [review]

**Pull request merged**